src/ansys/motorcad/core/geometry.py

```diff
@@ -561,17 +565,43 @@ def round_corner(self, corner_coordinate, radius):
         half_chord = radius * sin(radians(arc_angle) / 2)
         distance = abs(half_chord / (sin(radians(corner_angle) / 2)))
 
+        # check that distances by which the adjacent entities will be shortened are less than  the
+        # lengths of the adjacent entities.
         for index in range(len(adj_entities)):
             j = adj_entities[index]
+            if j.length < distance:
+                raise Exception(
+                    "Corner radius too large for these entities. "
```

Comment:
```suggestion
                    "Corner radius is too large for these entities. "
```
The string should have a verb.